### DISCUSSION TITLE
How to launch this kafka-ui in debug mode

### DISCUSSION THREAD

**Discussion by ramamanoj142000** [provectus/kafka-ui#1757] [Q&A] [answered]
How to launch this kafka-ui in debug mode.
I'm using Intellij for enhancement purpose so I can contribute 

[ERROR] Failed to execute goal on project kafka-ui-api: Could not resolve dependencies for project com.provectus:kafka-ui-api:jar:0.0.1-SNAPSHOT: Failed to collect dependencies at com.provectus:kafka-ui-contract:jar:0.0.1-SNAPSHOT: Failed to read artifact descriptor for com.provectus:kafka-ui-contract:jar:0.0.1-SNAPSHOT: com.provectus:kafka-ui:pom:${buildVersion} was not found in https://packages.confluent.io/maven/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of confluent has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[m_645600@AMBUS108329 15:31:47 ~/github/kafka-service-ui/kafka-ui-api]$ ~/apache-maven-3.8.3/bin/mvn spring-boot:run -Dspring-boot.run.jvmArguments="-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=5005"

**Comment by Haarolean** [comment, marked as answer]
Hey, you should build the whole app first.
From the project root: `mvn -Dmaven.test.skip=true clean install`.
After that you can run `spring-boot:run`
